vagrant/Vagrantfile

```diff
@@ -63,35 +31,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/home/vagrant/opt/monorail/"
         end
 
-        if ENV['TEST_GROUP']		
-          if ENV['TEST_GROUP'] =~  /esxi-5-5-min-install.v2.0.test/ || ENV['TEST_GROUP'] =~  /esxi-5-5-max-install.v2.0.test/		
```

Comment:
per @changev , seems those dhcpd.conf no necessarily to be customized. all test will pass with default dhcpd.conf in RackHD/packer folder